### PR TITLE
Add consistency check strategy to WAN reference config [CN-1129]

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

File: protocol-definitions/MC.yaml [hunk 1]
```diff
@@ -643,6 +643,14 @@ methods:
             0 - DISCARD_AFTER_MUTATION
             1 - THROW_EXCEPTION
             2 - THROW_EXCEPTION_ONLY_IF_REPLICATION_ACTIVE
+        - name: consistencyCheckStrategy
+          type: byte
```
[the remark this answers is not in the document]
Comment (SeriyBg) [reply protocol-definitions/MC.yaml:647]: Well, I followed the Hazelcast config type, which is also `byte`. I assume we don't exclude the addition of a new consistency check strategy in the future, and with `bool` it won't be scalable

**Review by ihsandemir — APPROVED** [review]

File: protocol-definitions/MC.yaml [hunk 2]
```diff
@@ -643,6 +643,14 @@ methods:
...
+          nullable: false
```
Comment (yuce) [inline protocol-definitions/MC.yaml:648]: @ihsandemir  Can't remember the exact rules here, but wouldn't it be better to mark this as `nullable`?

Comment (ihsandemir) [reply protocol-definitions/MC.yaml:648]: we do not mark primitives as nullable.

**Review by yuce — APPROVED** [review]
Looks good!

**Pull request merged**